stdlib/public/Platform/Darwin.swift

```diff
@@ -23,3 +23,9 @@ public let M_PI_2 = Double.pi / 2
 
 @available(swift, deprecated: 3.0, message: "Please use 'Double.pi / 4' or '.pi / 4' to get the value of correct type and avoid casting.")
 public let M_PI_4 = Double.pi / 4
+
+@available(swift, deprecated: 3.0, message: "Please use '2.squareRoot()'.")
+public let M_SQRT2 = 2.squareRoot()
```

Comment:
Would it be cleaner to use `sqrt(2)` instead here? (The free function `sqrt` is defined by Platform; it's possibly less Swifty but more familiar to people.)